provisioning/virtual-machines/ansible/roles/crownlabs/templates/cloud.cfg

```diff
@@ -42,7 +39,7 @@ cloud_config_modules:
 # this can be used by upstart jobs for 'start on cloud-config'.
  - emit_upstart
 # - snap
-# - ssh-import-id
+ - ssh-import-id
```

Comment:
Would it be possible to add a small comment for each line, telling why are the above modules needed?